llama_index/llms/openai.py

```diff
@@ -107,13 +107,16 @@ def class_name(cls) -> str:
     def _get_context_window(self) -> int:
         return openai_modelname_to_contextsize(self._get_model_name())
 
+    def _get_is_function_calling_model(self):
+        return (is_function_calling_model(self._get_model_name()),)
```

Comment:
```suggestion
    def _get_is_function_calling_model(self) -> bool:
        return is_function_calling_model(self._get_model_name())
```

This would have returned `tuple[bool]`, not `bool`.